tests/test_common_features.py

```diff
@@ -116,6 +117,12 @@ def create_test_layout(app_name, component_base, **kwargs):
     )
 
 
+def clear_field(web_elem):
+    """Analog to web_elem.clear() which does not work with Chrome currently."""
```

Comment:
```suggestion
    """Do the same as web_elem.clear() but work with Chrome."""
```
(Would be "analog of" anyway.)